### PR TITLE
Adds local and server search

### PR DESCRIPTION
**Pull request by HeDo88TH**
Closes https://github.com/DroneDB/Hub/issues/43

### CODE REVIEW

**Comment by HeDo88TH** [comment]
20bffea Closes https://github.com/DroneDB/Hub/issues/45

File: components/Explorer.vue [hunk 1]
```diff
@@ -96,6 +103,10 @@ export default {
         this.lazyLoadThumbs();
     },
     methods: {
+        filterFiles: function() {
+            //if (this.selectedFiles.length > 0) this.selectedFiles.forEach(f => f.selected = false);
+            return (this.filter == null || this.filter.length == 0) ? this.files : this.files.filter(i => i.entry.path.includes(this.filter));
```
Comment (pierotofy) [inline components/Explorer.vue:108]: Suggestion: use `computed` properties. https://vuejs.org/v2/guide/computed.html

Comment (pierotofy) [reply components/Explorer.vue:108]: Otherwise it will not be reactive.

Comment (HeDo88TH) [reply components/Explorer.vue]: Addressed in https://github.com/DroneDB/CommonUI/pull/10/commits/52cd72025176c39901429590606ebadf095df700

**Comment by pierotofy** [comment]
- Make sure to push your changes to `ddb_js` (https://github.com/DroneDB/ddb.js) (missing code)
 - Could the search be interactive (e.g. start searching after a small delay without having to press the search button)?
 - Text alignment for the search box should be left.
 - There should be no space to the left of the textbox. If you need an "X" (clear) button, it should probably be placed as an overlay.
 - 
image

**Comment by HeDo88TH** [comment]
> * Make sure to push your changes to `ddb_js` (https://github.com/DroneDB/ddb.js) (missing `ddb.searchEntries` code)
> * Could the search be interactive (e.g. start searching after a small delay without having to press the search button)?
> * Text alignment for the search box should be left.
> * There should be no space to the left of the textbox. If you need an "X" (clear) button, it should probably be placed as an overlay, to the right.
> 
> image

All done! 🚀

**Pull request merged**